decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -96,8 +96,9 @@ def smtp_environment
       end
 
       def install_decidim_webpacker
-        # Copy CSS variables template file
+        # Copy CSS files
         copy_file "decidim_application.scss", "app/packs/stylesheets/decidim/decidim_application.scss"
+        copy_file "_decidim-settings.scss", "app/packs/stylesheets/decidim/_decidim-settings.scss"
```

Comment:
I'm wondering if we should copy this file to the application by default.

I would be personally fine with that (since it's needed in all cases for us) but the current core does not automatically copy this file, which is why I wonder if it should be done.

Reply:
Right, I was copying it for simplicity for developers to start overriding vars (I think most of the Rails apps use this feature to change colors). What do you think @mrcasals ?